prow.sh

```diff
@@ -1066,15 +1066,21 @@ main () {
                 docker tag "$i:latest" "$i:csiprow" || die "tagging the locally built container image for $i failed"
             done
 
-            if [ -e deploy/kubernetes/rbac.yaml ]; then
+            if [ -e "deploy/kubernetes/rbac.yaml" ] || [ -e "deploy/kubernetes/csi-snapshotter/rbac-csi-snapshotter.yaml" ]; then
```

Comment:
This is still odd. Why do you need quotation marks here? It shouldn't make any difference whatsoever.

```
/tmp/empty$ touch foobar
/tmp/empty$ if [ -e foobar ]; then echo foobar exists; fi
foobar exists
/tmp/empty$ if [ -e "foobar" ]; then echo foobar exists; fi
foobar exists
```

Reply:
my mistake, I was updating this in my release-tools repo and testing in the sub-tree snapshotter version. You're right, it does not make a difference. Will update this too.